google-cloud-tools-plugin/src/com/google/gct/idea/CloudToolsPluginInitializationComponent.java

```diff
@@ -50,6 +52,9 @@ public void initComponent() {
           .registerExtension(
               ConfigurationType.CONFIGURATION_TYPE_EP, new CloudDebugConfigType());
     }
+    if (pluginInfoService.shouldEnable(GctFeature.MANAGEDVM)) {
+      pluginConfigurationService.registerExtension(ServerType.EP_NAME, new ManagedVmCloudType());
```

Comment:
In order to create a deployment configuration when trying to deploy to MVM I had to create a `DeployToServerConfigurationType` extension with name `CONFIGURATION_TYPE_EP` here in the initialization and pass the `ManangedVmCloudType` to it. 

IJ 15.0.3 now throws an exception if this isn't set up (instead of previously returning null): see `DeployToServerConfigurationRegsitrar.getDeployConfigurationType`.